Polygon.js

```diff
@@ -0,0 +1,85 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?://(?:www\\.)polygon\\.com",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-12-05 20:54:17"
+}
+
+// http://en.wikipedia.org/wiki/User_talk:Czar
+
+function detectWeb(doc, url) {
+	if (doc.title.match("| Polygon")) {
+ 		return "webpage";
+ 	}
+	
+	return null;
+}
+
+function scrape(doc, url) {
+	var namespace = doc.documentElement.namespaceURI; 
+	var nsResolver = namespace ? function(prefix) {
+		if (prefix == 'x') return namespace; else return null; 
+		} : null; 
```

Comment:
don't use the namespace declaration. If you really want to use doc.evaluate, simply replace nsResolver with null (which is all this function does for any regular html page anyway). But more below.